### ISSUE TITLE
Support scrolling the terminal

### ISSUE THREAD

**Issue by joshka** [charmbracelet/vhs#404]
**Is your feature request related to a problem? Please describe.**
Let's say we have a command that outputs a bunch of data to the terminal, some of which causes the terminal to scroll. I'd like some way to cause the terminal to scroll up to see the previous data in the gif.

My specific use case for this is in Ratatui (a rust TUI lib) testing that we can insert information before the current viewport (writable part of the screen for an app), and that this is in the correct position after the draw.

**Describe the solution you'd like**
Two commands ScrollUp N and ScrollDown N that scroll by a number of rows

**Describe alternatives you've considered**
I'm not sure if there is a generic way to do this with keyboard shortcuts (most apps will not pass PageUp/PageDown to the terminal in a way that's useful)

**Additional context**
Looking for less manual ways to test https://github.com/ratatui-org/ratatui/pull/596

**Comment by maaslalani** [comment]
Yes, we should certainly have this as well as other mouse controls. One thing I would change about the API is to introduce negative numbers as a way to scroll up lines (possibly):

```elixir
Scroll 5 # scroll down 5 lines
Scroll -5 # scroll up 5 lines
```

**Comment by joshka** [comment]
Makes sense

**Comment by joshka-oai** [comment]
I decided not to implement this as Scroll +/- for consistency with Up/Down PageUp/PageDown
